src/main.go

```diff
@@ -105,7 +105,8 @@ under GNU/GPLv2 license.`+"\n", version)
 	fmt.Println(gl.GoStr(gl.GetString(gl.VERSION)))
 
 	if debugGL {
-		gl.DebugMessageCallback(debugCb, unsafe.Pointer(nil))
+		// still causes crash so commented out:
+		//gl.DebugMessageCallback(debugCb, unsafe.Pointer(nil))
```

Comment:
As I said in the comment, I think this is an important feature because many GL calls can and will have something to say in case of problems, however I understand that a fix might need changes in the go-gl dependency :disappointed: